### DISCUSSION TITLE
Fault Codes

### DISCUSSION THREAD

**Discussion by JDC762** [slipx06/sunsynk-power-flow-card#546] [Sunsynk/Deye]
Is there a way to show the fault code that is displayed on the Sunsynk monitor on the HA Flow Card.  It shows ON OFF or say W04 as a fault.
I have looked at the Entities and I can't see a fault code entity; so this might well be the starting point.

**Comment by lightzhuk** [comment]
It would be really cool to support more error notification options

**Comment by lightzhuk** [comment]
![image](https://github.com/user-attachments/assets/25d92907-2e4c-470e-b540-e440fbc3e601)
